src/EnergyPlus/UnitarySystem.cc

```diff
@@ -1886,6 +1886,44 @@ namespace UnitarySystems {
 
             SizingMethod = DataHVACGlobals::SystemAirflowSizing;
 
+            MSHPIndex = this->m_DesignSpecMSHPIndex;
+            if (MSHPIndex > -1) {
+                // set no load air flow ratio local var
+                Real64 NoLoadCoolingAirFlowRateRatio = 1.0;
+                Real64 NoLoadHeatingAirFlowRateRatio = 1.0;
+                if (designSpecMSHP[MSHPIndex].coolingVolFlowRatio[0] == DataSizing::AutoSize) {
+                    NoLoadCoolingAirFlowRateRatio = 1.0 / double(designSpecMSHP[MSHPIndex].numOfSpeedCooling);
+                } else {
+                    NoLoadCoolingAirFlowRateRatio = min(this->m_NoLoadAirFlowRateRatio, designSpecMSHP[MSHPIndex].coolingVolFlowRatio[0]);
```

Comment:
here you are not allowing the user to choose the no load air flow rate. Should probably get rid of this and instead use a warning to the user.

Reply:
Oh, wait, what is the value of this->m_NoLoadAirFlowRateRatio here?

Reply:
At this point, the value of  this ->m_NoLoadAirFLowRateRatio is initialized to 1.0 if user left the field blank; otherwise, it may have any user specified value. I will modify it as follows to user input if it smaller than the autosized min flow fraction:
```
                if (designSpecMSHP[MSHPIndex].coolingVolFlowRatio[0] == DataSizing::AutoSize) {
                    NoLoadCoolingAirFlowRateRatio = min(this->m_NoLoadAirFlowRateRatio, 1.0 / double(designSpecMSHP[MSHPIndex].numOfSpeedCooling));
                } else {
                    NoLoadCoolingAirFlowRateRatio = min(this->m_NoLoadAirFlowRateRatio, designSpecMSHP[MSHPIndex].coolingVolFlowRatio[0]);
                }
                if (designSpecMSHP[MSHPIndex].heatingVolFlowRatio[0] == DataSizing::AutoSize) {
                    NoLoadHeatingAirFlowRateRatio = min(this->m_NoLoadAirFlowRateRatio, 1.0 / double(designSpecMSHP[MSHPIndex].numOfSpeedHeating));
                } else {
                    NoLoadHeatingAirFlowRateRatio = min(this->m_NoLoadAirFlowRateRatio, designSpecMSHP[MSHPIndex].heatingVolFlowRatio[0]);
                }
                this->m_NoLoadAirFlowRateRatio = min(NoLoadCoolingAirFlowRateRatio, NoLoadHeatingAirFlowRateRatio);
```

Reply:
This looks good. There could be autosized or a value in this->m_NoLoadAirFlowRateRatio. So with this method the ratio used will be <= minimum flow. That sounds like it meets the issue requirement. Make sure the docs say this.

Reply:
I see, this field can't be autosized, so it's either 1 or a value < 1. This still works as a minimum of either this field or the cooling/heating air flow ratio.